lib/src/utils.dart

```diff
@@ -92,6 +92,13 @@ void logDependencyInfractions(String infraction, Iterable<String> dependencies)
   logger.warning([infraction, bulletItems(sortedDependencies), ''].join('\n'));
 }
 
+/// Logs a info with the given [info] and lists all of the given
+/// [dependencies] under that.
+void logDependencyInfo(String info, Iterable<String> dependencies) {
+  final sortedDependencies = dependencies.toList()..sort();
+  logger.info([info, bulletItems(sortedDependencies), ''].join('\n'));
+}
```

Comment:
We could probably just generalize the `logDependencyInfractions()` method above and have it take an optional log `Level`.